specification/app/data-plane/Microsoft.App.DynamicSessions/stable/2024-09-01/DynamicSessions.json

```diff
@@ -0,0 +1,884 @@
+        "values": [
+          {
+            "name": "inline",
+            "value": "Inline",
+            "description": "Inline."
+          },
+          {
+            "name": "inlineBase64",
+            "value": "InlineBase64",
+            "description": "InlineBase64."
+          },
+          {
+            "name": "inlineText",
+            "value": "InlineText",
+            "description": "InlineText."
+          }
+        ]
+      }
+    },
+    "ExecutionType": {
+      "type": "string",
+      "description": "Execution Type",
+      "enum": [
+        "Synchronous",
+        "Asynchronous"
+      ],
+      "x-ms-enum": {
+        "name": "ExecutionType",
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "synchronous",
+            "value": "Synchronous",
+            "description": "Synchronous"
+          },
+          {
+            "name": "asynchronous",
+            "value": "Asynchronous",
+            "description": "Asynchronous"
+          }
+        ]
+      }
+    },
+    "MultiArrayStatus": {
+      "type": "string",
+      "description": "Multi array status.",
+      "enum": [
+        "MultiArrayEnabled",
+        "MultiArrayDisabled"
+      ],
+      "x-ms-enum": {
+        "name": "MultiArrayStatus",
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "multiArrayEnabled",
+            "value": "MultiArrayEnabled",
+            "description": "Multi array enabled"
+          },
+          {
+            "name": "multiArrayDisabled",
+            "value": "MultiArrayDisabled",
+            "description": "Multi array disabled."
+          }
+        ]
+      }
+    },
+    "NetworkStatus": {
+      "type": "string",
+      "description": "Network status.",
+      "enum": [
+        "EgressEnabled",
+        "EgressDisabled"
+      ],
+      "x-ms-enum": {
+        "name": "NetworkStatus",
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "egressEnabled",
+            "value": "EgressEnabled",
+            "description": "Session egress enabled."
+          },
+          {
+            "name": "egressDisabled",
+            "value": "EgressDisabled",
+            "description": "Session egress disabled."
+          }
+        ]
+      }
+    },
+    "PagedSessionResourceFile": {
+      "type": "object",
+      "description": "Paged collection of SessionResourceFile items",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "The SessionResourceFile items on this page",
+          "items": {
+            "$ref": "#/definitions/SessionResourceFile"
+          },
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "type": "string",
+          "format": "uri",
+          "description": "The link to the next page of items"
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "SessionCodeExecutionAsyncResponse": {
```

Comment:
I don't see any references to this schema. Can this be removed?

Reply:
I think this is auto generated by TypeSpec, the actual usage is break down to SessionCodeExecutionResponse and two headers in `CodeExecution_ExecuteCode`